src/components/graph/graph.coords.js

```diff
@@ -0,0 +1,333 @@
+import CONST from "./graph.const";
+import { logWarning } from "../../utils";
+
+/**
+ * Computes the normalized vector from a vector.
+ * @param {Object} vector a 2D vector with x and y components
```

Comment:
The convention for documentation is as follows: **variable - description.**
Here it would be like : 
`vector - a 2D vector with x and y components.`

Can you correct this for all doc parameters?